openquake/server/tests/views_test.py

```diff
@@ -103,26 +106,29 @@ def test_get(self):
         expected_content = [
             {u'url': u'http://www.openquake.org/v1/calc/1',
              u'status': u'executing',
+             u'job_type': u'risk',
              u'description': u'description 1',
              u'id': 1},
             {u'url': u'http://www.openquake.org/v1/calc/2',
              u'status': u'pre_executing',
+             u'job_type': u'risk',
              u'description': u'description 2',
              u'id': 2},
             {u'url': u'http://www.openquake.org/v1/calc/3',
              u'status': u'complete',
+             u'job_type': u'risk',
              u'description': u'description e',
              u'id': 3},
         ]
         with mock.patch('openquake.server.views._get_calcs') as grc:
             grc.return_value = [
-                (1, 'executing', 'description 1'),
-                (2, 'pre_executing', 'description 2'),
-                (3, 'complete', 'description e'),
+                (1, 'executing', 'risk', 'description 1'),
+                (2, 'pre_executing', 'risk', 'description 2'),
+                (3, 'complete', 'risk', 'description e'),
             ]
-            request = self.factory.get('/v1/calc/risk/')
+            request = self.factory.get('/v1/calc/list?job_type=risk')
             request.META['HTTP_HOST'] = 'www.openquake.org'
-            response = views.calc(request, 'risk')
+            response = views.calc(request)
 
             self.assertEqual(200, response.status_code)
             self.assertEqual(expected_content, json.loads(response.content))
```

Comment:
The problem is that current tests are mocking completely the calc method, so they do not check nothing of it, including the `relevant` attribute. The current tests are dangerous more than useless and I plan to remove them ( https://bugs.launchpad.net/oq-engine/+bug/1393733). This will be done in a separated pull request, though.